### PR TITLE
Test new hash table

### PR DESCRIPTION
**Pull request by DumDereDum**
### Pull Request Readiness Checklist

New Hash table for GPU version of code

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [ ] I agree to contribute to the project under Apache 2 License.
- [ ] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [ ] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: modules/rgbd/src/hash_tsdf.hpp [hunk 1]
```diff
@@ -31,6 +31,7 @@ class HashTSDFVolume : public Volume
     float truncDist;
     float truncateThreshold;
     int volumeUnitResolution;
+    Point3i volumeUnitResolutions;
```
Comment (savuor) [inline modules/rgbd/src/hash_tsdf.hpp:34]: Please update the code with recent commits; this is from previous code version

File: modules/rgbd/src/hash_tsdf.cpp [hunk 2]
```diff
@@ -31,6 +31,7 @@ HashTSDFVolume::HashTSDFVolume(float _voxelSize, cv::Matx44f _pose, float _rayca
     maxWeight(_maxWeight),
     truncateThreshold(_truncateThreshold),
     volumeUnitResolution(_volumeUnitRes),
+    volumeUnitResolutions(_volumeUnitRes, _volumeUnitRes, _volumeUnitRes),
```
Comment (savuor) [inline modules/rgbd/src/hash_tsdf.cpp:34]: The same: please update it with code changes

File: modules/rgbd/src/hash_tsdf.hpp (hunk 1 above)
Comment (DumDereDum) [reply modules/rgbd/src/hash_tsdf.hpp]: done

File: modules/rgbd/src/hash_tsdf.cpp (hunk 2 above)
Comment (DumDereDum) [reply modules/rgbd/src/hash_tsdf.cpp]: done

**Comment by DumDereDum** [comment]
PR was exhausted
I am going to create a new PR

**Pull request closed without merging**